specification/imds/data-plane/readme.md

```diff
@@ -0,0 +1,74 @@
+# Instance Metadata Service
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Instance Metadata Service.
+
+## Getting Started
+To build the SDKs for Instance Metadata Service, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+> `autorest readme.md`
+
+To see additional help and options, run:
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+These are the global settings for the Instance Metadata Service API.
+
+``` yaml
+openapi-type: data-plane
+azure-arm: true
+tag: package-2018-05
+input-file:
+- Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json
+output-folder: ./Generated
+```
+
+## Suppression
+ ``` yaml
+ directive:
+   - suppress: DefinitionsPropertiesNamesCamelCase
+     reason: The following properties follow the Oath2 spec, which does not use camelCase.
+     from: Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json
+     where:
+       - $.definitions.IdentityTokenResponse.properties.access_token
+       - $.definitions.IdentityTokenResponse.properties.expires_in
+       - $.definitions.IdentityTokenResponse.properties.expires_on
+       - $.definitions.IdentityTokenResponse.properties.ext_expires_in
+       - $.definitions.IdentityTokenResponse.properties.not_before
+       - $.definitions.IdentityTokenResponse.properties.resource
+       - $.definitions.IdentityTokenResponse.properties.token_type
+       - $.definitions.IdentityTokenResponse.properties.client_id
+       - $.definitions.IdentityTokenResponse.properties.object_id
+       - $.definitions.IdentityTokenResponse.properties.msi_res_id
+       - $.definitions.IdentityErrorResponse.properties.error_description
+	   
```

Comment:
Can you remove the empty line here before the  \`\`\`?

I guess the linter error is failing for one property because of that, even though it has been added to the suppression list
-- | -- | --
R3016 - DefinitionsPropertiesNamesCamelCase | 2018-10-01/​imds.json:675 | Property named: 'error_description', for definition: 'IdentityErrorResponse' must follow camelCase style. Example: 'error_description'.
-- | -- | --